templates/hello_world_ts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "my-awesome-actor",
+    "version": "0.0.1",
+    "description": "This is a boilerplate of an Apify actor.",
+    "dependencies": {
+        "apify": "^0.19.1"
+    },
+    "devDependencies": {
+        "@types/node": "^13.7.7",
+        "@types/cheerio": "^0.22.16",
+        "@types/puppeteer": "^2.0.1",
+        "typescript": "^3.8.3"
+    },
+    "scripts": {
+        "build": "tsc -p tsconfig.json",
+        "start": "node lib/main.js",
```

Comment:
I guess you need to update npm start command because you need to build files the first.

Reply:
yeah, usually you need to build locally (doing `npm run build` before pushing), so you don't add devDependencies on the image. the `tsconfig.json` generates the files to the `lib` folder